javaparser-core/src/main/java/com/github/javaparser/TokenTypes.java

```diff
@@ -62,13 +76,16 @@ public static boolean isSpaceOrTab(int tokenType) {
         }
     }
 
-    public static boolean isComment(int tokenType) {
-        return tokenType == SINGLE_LINE_COMMENT
-                || tokenType == MULTI_LINE_COMMENT
-                || tokenType == JAVA_DOC_COMMENT;
+    public static boolean isComment(int kind) {
+        return getCategory(kind)== JavaToken.Category.COMMENT;
     }
 
+    /** @deprecated use eolTokenKind */
     public static int eolToken() {
```

Comment:
The people who decided that that annotation cannot have a reason field should be spanked.